src/main/java/uk/gov/companieshouse/overseasentitiesapi/converter/OverseasEntitySubmissionDaoConverter.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.companieshouse.overseasentitiesapi.converter;
+
+import org.bson.Document;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.ReadingConverter;
+import org.springframework.data.mongodb.core.convert.MongoConverter;
+import uk.gov.companieshouse.overseasentitiesapi.model.dao.OverseasEntitySubmissionDao;
+
+
+@ReadingConverter
+public class OverseasEntitySubmissionDaoConverter implements Converter<Document, OverseasEntitySubmissionDao> {
+
+    private final MongoConverter defaultMongoConverter;
+
+    public OverseasEntitySubmissionDaoConverter(MongoConverter defaultMongoConverter) {
+        this.defaultMongoConverter = defaultMongoConverter;
+    }
+
+    @Override
+    public OverseasEntitySubmissionDao convert(final Document document) {
+
+        System.out.println("\n\n**************** Document: \n\n\n");
```

Comment:
Replace System.outs with LOG library calls.

Reply:
it was more of a debug line just to see if the new converter was being used, i assumed it would be taken out when the converter gets completed. I don't want to log anything specific here so i'll just remove it.